architecture/core_concepts/routes.adoc

```diff
@@ -314,6 +320,47 @@ they are unique on the machine. These ports will not be exposed externally.
 
 The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
 
+[[load-balancing]]
+== Load-balancing Strategy
+
+When a route has multiple endpoints, HAProxy distributes requests to the route
+among the endpoints based on the selected load-balancing strategy. This only
+applies when no persistence information is available, or when a connection is
+redispatched to another server. See the discussion on
+xref:routes-sticky-sessions[Sticky Sessions]
+for more details. The strategy my one of the following:
+
+- `*roundrobin*`  Each server is used in turns, according to their weights.
+This is the smoothest and fairest algorithm when the server's
+processing time remains equally distributed. This algorithm
+is dynamic, which means that server weights may be adjusted
+on the fly for slow starts for instance. It is limited by
+design to 4 active servers per route.
+- `*leastconn*`   The server with the lowest number of connections receives the
+connection. Round-robin is performed within groups of servers
+of the same load to ensure that all servers will be used. Use
+of this algorithm is recommended where very long sessions are
+expected, such as LDAP, SQL, TSE, etc... but is not very well
+suited for protocols using short sessions such as HTTP. This
+algorithm is dynamic, which means that server weights may be
```

Comment:
This is from the haproxy docs. round robin among the servers with the same number of connections.
The weights can apply to the group that is in the round robin mix. So, why do we default to this?